app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -13,6 +13,8 @@ def initialize(name, type, sandbox, build, role:, editable: false)
     # Make sure tree_state doesn't hold on to old data between requests
     TreeState.new(sandbox).remove_tree(name)
 
+    MiqProductFeature.invalidate_caches # clear cash because of creating/updating/deleleting tenant product features
```

Comment:
cash != cache ;)

Reply:
🤑

Reply:
:rofl: :rofl: :rofl: :rofl: this made my day